docs/source/api/fpsemi-examples.rst

```diff
@@ -0,0 +1,154 @@
+.. Copyright (c) 2022, J. D. Mitchell
+
+   Distributed under the terms of the GPL license version 3.
+
+   The full license is in the file LICENSE, distributed with this software.
+
+   This file only exists because Breathe always displays all members when
+   documenting a namespace, and this is nicer for now.
+
+.. cpp:namespace:: libsemigroups
+
+Examples
+--------------------
+
+Defined in ``present.hpp`` and ``knuth-bendix.hpp``.
+
+This page contains the documentation for various helper functions for
+manipulating :cpp:type:`Presentation` objects. All such functions are contained in
+the namespace ``presentation``.
+
+These functions are available in the ``present.hpp`` file except
+:cpp:any:`redundant_rule` which is in ``knuth-bendix.hpp``.
+
+Presentations from the following sources are implemented: :cite:`Gay1999aa`; :cite:`Godelle2009aa`; :cite:`East2022aa`; :cite:`Maltcev2007aa`;
+:cite:`Arthur2000aa`; :cite:`Kudryavtseva2006aa`; :cite:`Burnside2012aa`; :cite:`Ganyushkin2009aa`; :cite:`Cassaigne2001aa`; :cite:`Campbell1994aa`;
+:cite:`Abram2022aa`; :cite:`Easdown2007aa`; :cite:`FitzGerald2003aa`; :cite:`East2011aa`; :cite:`Ayik2000aa`; :cite:`Ruskuc1995aa`; :cite:`Aizenstat1958aa`;
+:cite:`Coxeter1979aa`; :cite:`Knuth1970aa`.
+
+Contents
+~~~~~~~~
+
+.. cpp:namespace:: libsemigroups
+
+.. list-table::
+   :widths: 50 50
+   :header-rows: 0
+
+   * - :cpp:any:`symmetric_group`
+     - Construct a presentation for the symmetric group.
```

Comment:
```suggestion
     - Presentation for the symmetric group.
```
Not sure if this is any better, but maybe remove the "Construct a" from each line here?